roles/module_openapi_cloud/templates/generator.py

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+import copy
+import re
+from typing import Iterable, List, Dict
+from gouttelette.utils import (
```

Comment:
Same as above. Would these get updated once your next PR is merged?

Reply:
yes, as discussed this file will move into plugin_utils.